forge-gui/res/tokenscripts/b_0_0_zombie_army.txt

```diff
@@ -0,0 +1,6 @@
+Name:Zombie Army Token
```

Comment:
so we need one for each type now?
what about _Lazotep Sliver_?

Reply:
Yea, but these scripts should already have been unique since they have different names.